src/FlapperUniV2.sol

```diff
@@ -36,6 +36,7 @@ interface GemLike {
     function decimals() external view returns (uint8);
     function approve(address, uint256) external;
     function transfer(address, uint256) external;
+    function balanceOf(address) external view returns (uint256);
```

Comment:
nit: I'd put it above to group with the other `view` as in the rest of interfaces.

Reply:
Done.